### PR TITLE
Add pyRingGW package

### PR DESCRIPTION
**Pull request by GCArullo**
@conda-forge/help-python, ready for review!

@conda-forge-admin, please ping @conda-forge/help-python

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.

I agree to be listed as a maintainer.

### CODE REVIEW

**Comment by GCArullo** [comment]
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

**Comment by GCArullo** [comment]
I am OK with being listed as a maintainer.

File: recipes/pyRingGW/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  host:
+    - python
+    - cython
+    - numpy
+    - pip
+  run:
+    - python
+    - corner
+    - cpnest
+    - cython
```
Comment (carterbox) [inline recipes/pyRingGW/meta.yaml:31]: Why do you need cython at runtime? Typically once the cython files have been translated to c and compiled into shared objects, cython is not needed anymore.

File: recipes/pyRingGW/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  entry_points:
+    - pyRing = pyRing.pyRing:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
```
Comment (carterbox) [inline recipes/pyRingGW/meta.yaml:23]:
```suggestion
    - python
    - setuptools
```

File: recipes/pyRingGW/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyRingGW-{{ version }}.tar.gz
+  sha256: f354b2ee1104d09091c1ea1ca7f1d05ad08aa8f0fd1c09bc0a8605917a6b5ad6
+
+build:
+  skip: true  # [py<36]
```
Comment (carterbox) [inline recipes/pyRingGW/meta.yaml:13]:
```suggestion
  skip: true  # [py<36]
  skip: true  # [win]
```
Please skip Windows if it is not possible to build on Windows.

**Review by carterbox — CHANGES_REQUESTED** [review]

File: recipes/pyRingGW/meta.yaml (hunk 2 above)
Comment (GCArullo) [reply recipes/pyRingGW/meta.yaml:24]: Addressed [here](https://github.com/conda-forge/staged-recipes/pull/21577/commits/1ba3cf2305479c8cfb8b6c4dfff1d0263884cd1d).

File: recipes/pyRingGW/meta.yaml (hunk 3 above)
Comment (GCArullo) [reply recipes/pyRingGW/meta.yaml:13]: Addressed [here](https://github.com/conda-forge/staged-recipes/pull/21577/commits/1ba3cf2305479c8cfb8b6c4dfff1d0263884cd1d).

**Comment by carterbox** [comment]
You could have submitted the "art" package in this PR as a separate recipe. You still can submit a separate PR for the "art" package if you haven't done so already.

**Pull request merged**